README.md

```diff
@@ -1,4 +1,4 @@
-# witnet_wallet
+# myWitWallet
```

Comment:
@witnet/devs @witnet/brick-mortars `MyWitWallet` or `myWitWallet`?

Reply:
I prefer `myWitWallet`

Reply:
I think I prefer `myWitWallet` too. It does bring more attention to the `Wit` part.

Reply:
I prefer `MyWitWallet` because it seems rare to me that a proper noun doesn't start with a capital letter.